### PR TITLE
wi330 disaster recovery best practices incl. Openshift Gold

### PR DESCRIPTION
**Pull request by pdoc1111**
Implements # \<issue number\>

### CODE REVIEW

**Comment by marcellmueller** [comment]
These docs are looking amazing, nice job.

Just leaving a note since I don't believe we mentioned it but the PR is failing the cocogitto test for commit style, we are using conventional commits:
https://www.conventionalcommits.org/en/v1.0.0/

So just need to rename commit to start with likely `docs: `

File: docs/disasterRecovery/disasterRecovery.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+In case of a database corruption, crash or any other issue requiring us to restore the database to a previous state, the OpenShift `PostgresCluster` object allows us to restore the database from any point in time prior to the incident.
+
+---
+
+## Makefile target
+
+**Prerequisites:**
+
+- The ENVIRONMENT (dev,test,prod) and CIF_NAMESPACE_PREFIX, GGIRCS_NAMESPACE_PREFIX, CIIP_NAMESPACE_PREFIX environment variables are set
```
Comment (AntBush) [inline docs/disasterRecovery/disasterRecovery.md:11]: Need to update these for CCBC

File: docs/disasterRecovery/disasterRecovery.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+- The machine is logged in onto the cluster where the CIF app is deployed (`oc login ...`)
+
+**Restoring the DB:**
+
+To restore the database to any point in time, simply run:
+
+```
+make restore TARGET_TIMESTAMP="2022-05-24 14:20:00-07"
```
Comment (AntBush) [inline docs/disasterRecovery/disasterRecovery.md:19]: We don't have a `make restore` command in our makefile, would be valuable to add this in or update this with the manual steps

File: docs/disasterRecovery/point-in-time-recovery.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+The helm/patroni folder contains a fork of the patroni helm chart that was in the (now deprecated) helm charts incubator, and can be reused by other teams. The helm/cas-patroni](helm/cas-patroni) folder contains helm values used by the CAS team, as well as a job definition for the [cas-shelf` image, which provisions object storage buckets when WAL-G backups are enabled.
+
+# Installation
+Installation of the cas-postgres helm chart is simply done via the helm install command. See helm/cas-postgres/values.yaml for more information on the values that should be provided
+
+# Steps for a complete fresh install (will lose all data)
+helm delete the cas-ciip-portal release ex: helm delete --namespace wksv3k-dev cas-ciip-portal
+delete the cas-ciip-portal patroni pvcs's (can be done from the openshift ui under storage)
+delete the cas-ciip-portal patroni configmaps (can be done from the openshift ui under Resources -> Configmaps)
+the trigger-dag job may possibly need to be deleted as well. Look for the job in the openshift ui under Resources -> Other Resources, if you find the job, delete it.
```
Comment (AntBush) [inline docs/disasterRecovery/point-in-time-recovery.md:27]: Should be updated for CCBC, we don't have DAG/airflow in our project

File: docs/disasterRecovery/disasterRecovery.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+To allow restoring with the [helm chart](https://github.com/bcgov/cas-cif/tree/develop/chart/cas-cif), the bucket provisioning and database pre-upgrade jobs have been disabled directly in the chart when the restore option is enabled.
+A few more values need to be adjusted:
+
+To avoid running some jobs and airflow DAGs:
+
+```yaml
+# chart/cas-cif/values.yaml
+deploy-db:
+  enabled: false
+download-dags:
```
Comment (AntBush) [inline docs/disasterRecovery/disasterRecovery.md:48]: We should update this as well for CCBC, I don't believe these values are in our chart

**Review by AntBush — CHANGES_REQUESTED** [review]
Looks good! Appreciate the added docs for this, just needs some CCBC-specific updates

**Pull request closed without merging**